### PR TITLE
feat(pci.storages.databases): datatr-314 - add region option to fork

### PR DESCRIPTION
**Pull request by jperchoc**
DATATR-314

| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #DATATR-314
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

Add region selection in fork page

### :flags: Translations

- [x] CMT-4221

### CODE REVIEW

File: packages/manager/modules/pci/src/components/project/regions-list/regions-list.html [hunk 1]
```diff
@@ -32,6 +32,14 @@
                         data-disabled="$ctrl.disabled || $ctrl.constructor.isRegionDisabled(regions)"
                         data-required
                     >
+                        <oui-select-picker-picture
+                            data-ng-if="$ctrl.currentRegion.name === regions[0].microRegion.code"
```
Comment (darsene) [inline packages/manager/modules/pci/src/components/project/regions-list/regions-list.html:36]: What about if regions length > 1 ?

File: packages/manager/modules/pci/src/projects/project/storages/databases/components/flavors-list/flavors-list.html [hunk 2]
```diff
@@ -54,7 +54,7 @@
 
             <span
                 class="oui-badge oui-badge_s oui-badge_success"
-                data-ng-if="$ctrl.currentFlavor.id === flavor.id"
+                data-ng-if="$ctrl.currentFlavor.name === flavor.name"
```
Comment (darsene) [inline packages/manager/modules/pci/src/projects/project/storages/databases/components/flavors-list/flavors-list.html:57]: Why can't you keep `id` ?

File: packages/manager/modules/pci/src/components/project/regions-list/regions-list.html (hunk 1 above)
Comment (jperchoc) [reply packages/manager/modules/pci/src/components/project/regions-list/regions-list.html:36]: We are iterating through regions in the ng-repeat, so if regions.length < 1, this block should not exist.

File: packages/manager/modules/pci/src/projects/project/storages/databases/components/flavors-list/flavors-list.html (hunk 2 above)
Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/storages/databases/components/flavors-list/flavors-list.html:57]: The id is a concatenation of both the flavor name and the plan name. Here, we want to display the badge even if the plan is different.
For exemple, if the current flavor is a db-4 with a business plan, and we choose an enterprise plan for the fork, then we want to show the "current offer" badge next to the flavor with id `enterprise-db-4`.

**Review by darsene — DISMISSED** [review]

**Review by abullet33 — DISMISSED** [review]

**Review by anooparveti — DISMISSED** [review]

**Review by anooparveti — DISMISSED** [review]

**Review by sachinrameshn — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**